### PR TITLE
Definition for Notification

### PR DESCRIPTION
**Pull request by JeremyTubongbanua**
**- What I did**
Start on notification info.
Feel free to give feedback on this page. A little bit lost with this definition!

**- How I did it**
See files changed

**- How to verify it**
image

### CODE REVIEW

File: content/docs/reference/notification.md [hunk 1]
```diff
@@ -9,8 +9,55 @@ description: | # SEO Description of the page (Shows in google and atsign.dev sea
     Definition of Notificaton in the @platform
```
Comment (xavierchanth) [inline content/docs/reference/notification.md]: Clean up the lead and expand upon description please

File: content/docs/reference/notification.md [hunk 2]
```diff
@@ -9,8 +9,55 @@ description: | # SEO Description of the page (Shows in google and atsign.dev sea
...
+
+Read more [here](https://blog.atsign.dev/part-1-the-notify-verb-cko97bv8f00l5gws13umb0nvz).
+
+## @protocol
+
+The notify verb enables you to notify the atsign user of some data event.
+
+The follwing is the regex for the `notify` verb.
+
+`notify:((?<operation>update|delete):)?(ttl:(?<ttl>\d+):)?(ttb:(?<ttb>\d+):)?(ttr:(?<ttr>(-)?\d+):)?(ccd:(?<ccd>true|false):)?(@(?<forAtSign>[^@:\s]-)):(?<atKey>[^:]((?!:{2})[^@])+)@(?<atSign>[^@:\s]+)(:(?<value>.+))?`
```
Comment (xavierchanth) [inline content/docs/reference/notification.md:30]: Can you break down and explain this regex please, that will be more useful to readers than just providing it.

**Comment by JeremyTubongbanua** [comment]
I don't know if there's much more to say about **Notification**. We will dive more into code stuff in another section of the dev site.

Awaiting feedback for this page @XavierChanth @Jesseadriano10 @L1AMcB.

**Review by xavierchanth — CHANGES_REQUESTED** [review]

File: content/docs/reference/notification.md (hunk 2 above)
Comment (JeremyTubongbanua) [reply content/docs/reference/notification.md:30]: See 3c7f10af673287485c2422bdfc7556b9d58ef49e

Am stuck on the AtKey details regex part

Please assist on this part: 
image

Each regex snippet is missing argument details and an example

File: content/docs/reference/notification.md (hunk 1 above)
Comment (JeremyTubongbanua) [reply content/docs/reference/notification.md]: See 3c7f10af673287485c2422bdfc7556b9d58ef49e

File: content/docs/reference/notification.md (hunk 2 above)
Comment (JeremyTubongbanua) [reply content/docs/reference/notification.md:30]: @TylerTrott

Comment (xavierchanth) [reply content/docs/reference/notification.md:30]: See if parts of the doc I just sent you help @JeremyTubongbanua

**Comment by JeremyTubongbanua** [comment]
I think this is right? c3ed93e34877935109d024bcdee45ac6908fcc56

**Review by xavierchanth — APPROVED** [review]

**Pull request merged**